bedrock/firefox/views.py

```diff
@@ -549,7 +550,10 @@ def get_context_data(self, **kwargs):
 
         analytics_version = str(num_version) + channel
         entrypoint = 'mozilla.org-whatsnew' + analytics_version
-        campaign = 'whatsnew' + analytics_version
+        if version.startswith('79.'):
+            campaign = 'mozilla-vpn-v1-0-announcement'
```

Comment:
do we need to add a `channel` check in here to make sure only this WNP gets the custom param?